modules/ebi-metagenomics/interproscan/tests/main.nf.test

```diff
@@ -0,0 +1,43 @@
+// This krona/ktimporttext unit test file is converted to nf-test from an existing pytest test at this commit: https://github.com/nf-core/modules/commit/5e34754d42cd2d5d248ca8673c0a53cdf5624905
+// I have converted it because the standard way of writing nf-core unit tests now is to use nf-test
+// It is otherwise exactly the same unit test
+
+nextflow_process {
+
+    name "Test Process INTERPROSCAN"
+    script "../main.nf"
+    process "INTERPROSCAN"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "interproscan"
+
+    test("test_interproscan") {
+
+        when {
+            process {
+                """                
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file("${moduleDir}/tests/test.fasta", checkIfExists: true)
```

Comment:
let's move the test file (test.fasta) to a folder named 'fixtures'? Probably the interproscan_db as well? Keep the folder architecture more or less the same across modules.